docker/Dockerfile

```diff
@@ -57,11 +58,19 @@ RUN useradd -s /usr/sbin/nologin --create-home --password odoo odoo \
 COPY docker/msfonts.sh /tmp/msfonts.sh
 RUN chmod +x /tmp/msfonts.sh; sync; /tmp/msfonts.sh; rm -rf /tmp/msfonts.sh
 
+# Python
+ARG PYTHON_VERSION=3.8.10
+ENV PYENV_ROOT="$HOME/.pyenv"
+ENV PATH="${PYENV_ROOT}/shims:${PYENV_ROOT}/bin:${PATH}"
+RUN git clone --depth 1 -b v2.4.1 https://github.com/pyenv/pyenv.git "${PYENV_ROOT}" \
+    && pyenv install --verbose ${PYTHON_VERSION} \
+    && pyenv global ${PYTHON_VERSION}
+
 # pip dependencies
 COPY requirements.txt /tmp/requirements.txt
 RUN pip3 install --no-cache-dir setuptools --upgrade \
     && pip3 install --no-cache-dir --upgrade pip \
-    && pip3 install --no-cache-dir -r /tmp/requirements.txt \
+    && pip3 install --no-cache-dir python-magic==0.4.27 cairocffi==0.9.0 -r /tmp/requirements.txt \
```

Comment:
Would it make sense to add `python-magic` and `cairocffi` to the `requirements.txt` file?